source/app/powertabeditor.cpp

```diff
@@ -2425,6 +2446,11 @@ void PowerTabEditor::createCommands()
                               tr("Slide Out Of Upwards"),
                               "TabSymbols.SlideOutOf.Upwards", QKeySequence(),
                               Note::SlideOutOfUpwards);
+    
+    myFingerCommand = new Command(tr("Finger..."), "TabSymbols.Finger",
+                                  QKeySequence(tr("Shift+F")), this);
+    myFingerCommand->setCheckable(true);
+    connect(myFingerCommand, SIGNAL(triggered()), this, SLOT(editFingerLeft()));
```

Comment:
The code isn't consistent about this yet, but I'd prefer to use the new-style connections that are checked at compile time (one example is for myAddPlayerCommand) instead of the SLOT() syntax